06-single-hop-udp/task01.py

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python3
+# Copyright (C) 2018 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+import os
+
+PORT            = 1337
+COUNT           = 1000
+PAYLOAD_SIZE    = 1024
+DELAY           = 1000  # ms
+ERROR_TOLERANCE = 5     # %
+
+
+def task01(riotbase, runs=1):
+    os.chdir(os.path.join(riotbase, "tests/gnrc_udp"))
+    try:
+        exp = IoTLABExperiment("RIOT-release-test-06-01",
+                               [IoTLABNode(site="lille",
```

Comment:
I picked Lille for both experiments, because Grenoble is most often too noisy to get 1000 packets of >10 fragments through at such a low error tolerance (see https://github.com/RIOT-OS/Release-Specs/issues/53).